pkg/sdk/api/v1beta1/fluentd_types.go

```diff
@@ -60,8 +60,14 @@ type FluentdSpec struct {
 	Scaling              *FluentdScaling             `json:"scaling,omitempty"`
 	Workers              int32                       `json:"workers,omitempty"`
 	// +kubebuilder:validation:enum=fatal,error,warn,info,debug,trace
-	LogLevel             string `json:"logLevel,omitempty"`
-	PodPriorityClassName string `json:"podPriorityClassName,omitempty"`
+	LogLevel string `json:"logLevel,omitempty"`
+	// Ignore same log lines in seconds
+	// +docLink:"more info, https://docs.fluentd.org/deployment/logging#ignore_same_log_interval"
+	IgnoreSameLogInterval int32 `json:"ignoreSameLogInterval,omitempty"`
+	// Ignore repeated log lines in seconds
+	// +docLink:"more info, https://docs.fluentd.org/deployment/logging#ignore_repeated_log_interval"
+	IgnoreRepeatedLogInterval int32  `json:"ignoreRepeatedLogInterval,omitempty"`
+	PodPriorityClassName      string `json:"podPriorityClassName,omitempty"`
```

Comment:
Based on the docs this should be a string field to be able to set time intervals like "60s", "1m" ...